spec.md

```diff
@@ -167,6 +167,43 @@ merged are hosted in other folders.
 As some Compose file elements can both be expressed as single strings or complex objects, merges MUST apply to
 the expanded form.
 
+### Profiles
+
+Profiles allow to adjust the Compose application model for various usages and environments. A Compose
+implementation SHOULD allow user to define a set of active profiles. The exact mechanism is implementation
+specific and MAY include command line flags, environment variables, etc.
+
+All top-level elements support a `profiles` attribute to define a list of named profiles. Elements without 
+a `profiles` attribute set MUST always be enabled. The enclosing component MUST be ignored by compose
+implementation when none of the listed profiles match the active ones, until the target component is
+explicitly targeted by a command.
+
+References to other services (by `links`, `extends` or shared resource syntax `service:xxx`) MUST not
```

Comment:
See https://github.com/compose-spec/compose-spec/issues/97#issuecomment-685712571

Reply:
The term "shared resource syntax `service:xxx`" is not defined anywhere in the spec document.

Where can a newcomer learn what is it? It would be good if there was at least a hyperlink here.

Reply:
This is referring to system resources shareable between services or with the host, where you can specify the resource with the `service:[service name]` syntax, see [`network_mode`](https://github.com/compose-spec/compose-spec/blob/9015d1b3e0cd0504e38b9d22f8845068c7f22584/spec.md#network_mode) and [`ipc`](https://github.com/compose-spec/compose-spec/blob/9015d1b3e0cd0504e38b9d22f8845068c7f22584/spec.md#ipc) in the spec.

Reply:
Thanks for the clarification. If the document included these links to the spec that would be really helpful for other people, too. Do you think it can be added?

Reply:
Now I see that this snippet was already merged in https://github.com/compose-spec/compose-spec/pull/110/files#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R182.

My comment should really be placed there. The gist of it stands: could a link [network_mode](https://github.com/compose-spec/compose-spec/blob/9015d1b3e0cd0504e38b9d22f8845068c7f22584/spec.md#network_mode) and [ipc](https://github.com/compose-spec/compose-spec/blob/9015d1b3e0cd0504e38b9d22f8845068c7f22584/spec.md#ipc) be added in the document?